index.html

```diff
@@ -1,22 +1,30 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="ja">
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <link rel="stylesheet" href="style.css" type="text/css" />
-    <title>TerraceTechフロントエンドエンジニア養成所（修正済みです！）</title>
+    <title>TerraceTechフロントエンドエンジニア養成所</title>
   </head>
   <body>
     <div>
-      <ul id="js-list"></ul>
+      <ul id="js-list">
+      </ul>
     </div>
-    <script>
-      const ul = document.getElementById('js-list');
-      const li = document.createElement('li'); const a = document.createElement('a'); const img = document.createElement('img');
-      a.setAttribute("href","1.html"); img.setAttribute("src","bookmark.png"); img.setAttribute("alt","ブックマーク"); 
-      a.textContent = 'これです';
-      ul.appendChild(li);  li.appendChild(a); a.appendChild(img);
+    <script>    
+      var ul = document.getElementById('js-list');
```

Comment:
`var` is not recommended.
Let's use `const` .
But use `let` when a variable is reassign.

Reply:
Thank you for your comment.
I will fix it and work hard to brush it up.